add an option to limit the number of connections per port (#3)
Tecnativa/docker-whitelist: README.md

Set to `udp` to proxy in UDP mode.

### `MAX_CONNECTIONS`

Default: `100`

> Limits the maximum number of accepted connections at one time per port.

Reviewer: ```suggestion
Limits the maximum number of accepted connections at once per port.
```

> 

Reviewer 2: ```suggestion
You can set it to `4194304` to set it to unlimited (See https://stackoverflow.com/questions/6294133/maximum-pid-in-linux)
```

Author: I'll add a note matching this. however just to be clear this would probably need to be set on the docker host system (i'll investigate) and it is far from unlimited in the mathematical sense of numbers.

Reviewer 2: Ok :+1: 
Yes, that was only a suggestion, it can be phrased better :smile:

Set to `udp` to proxy in UDP mode.

### `MAX_CONNECTIONS`

Default: `100`

Limits the maximum number of accepted connections at once per port.

#### Setting "unlimited" connections

For each port and open connection a subprocess is spawned. Setting

> a number to high might make your host system unresponsive and prevent you from

Reviewer 2: ```suggestion
a number too high might make your host system unresponsive and prevent you from
```
